src/pages/workspaces/workspace/analysis/AppLauncher.js

```diff
@@ -27,9 +28,14 @@ const ApplicationLauncher = _.flow(
   })
 )(({
   name: workspaceName, sparkInterface, analysesData: { runtimes, refreshRuntimes },
-  application, workspace: { azureContext, workspace: { namespace, name, workspaceId, googleProject, bucketName } }
+  application, workspace: { azureContext, workspace: { workspaceId, googleProject, bucketName }, namespace }
 }, _ref) => {
-  const [busy, setBusy] = useState(true)
+  useEffect(() => {
+    Ajax().Metrics.captureEvent(Events.analysisLaunch,
+      { origin: 'appLauncher', source: application, application, workspaceName, namespace, cloudPlatform: googleProject ? 'GCP' : 'Azure' })
```

Comment:
Since workspace is de structured, I wasn't able to use `getCloudProviderFromWorkspace` when collecting cloudPlatform.

Should we change the de structuring to accommodate that check?